### DISCUSSION TITLE
Updating constantly

### DISCUSSION THREAD

**Discussion by saditude** [qdm12/ddns-updater#800] [General]
I have this set up to where it updates a cloudflare record, but it's updated it almost 1500 times although it hasnt changed. Is this normal? and if so is there a way to only update it when needed?
![Screenshot 2024-09-02 142803](https://github.com/user-attachments/assets/b136bdc8-d6b9-445d-bd6e-0e96bdd2e680)
I'm not very advanced in any way with coding of any sorts, so apologies if I don't understand immediately.

**Comment by qdm12** [comment]
You probably need to set the proxied field to true in your json configuration. Otherwise ddns-updater thinks the ip address of the A record doesn't match your public IP address, and tries to update it again.
